### PR TITLE
Avatar landing pages

### PR DESCRIPTION
**Pull request by netpro2k**
You can now link to custom avatars directly to share them with others.

This PR adds very basic avatar landing pages. More work can certainly be done on the design of these but the basic functionality is there. This requires accompanying changes from https://github.com/mozilla/reticulum/pull/183.

### CODE REVIEW

File: src/assets/stylesheets/avatar.scss [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+
+  :local(.logo) {
+    width: 150px;
+    position: absolute;
+    right: 32px;
+    bottom: 32px;
+    display: none;
+
+    @media (min-width: 500px) {
+      display: block;
```
Comment (brianpeiris) [inline src/assets/stylesheets/avatar.scss]: I think it's important that we fit the logo on the page even on mobile. Unlike the Scene page, this page makes very little sense without context, if you've never heard of Hubs.

File: src/avatar.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+<html>
+
+<head>
+    <!-- DO NOT REMOVE/EDIT THIS COMMENT - META_TAGS -->
+
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+    <link rel="shortcut icon" type="image/png" href="/favicon.ico">
+    <title>Scene | Hubs by Mozilla</title>
```
Comment (brianpeiris) [inline src/avatar.html]: nit
```suggestion
    <title>Avatar | Hubs by Mozilla</title>
```

File: src/avatar.js [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+import PropTypes from "prop-types";
+import classNames from "classnames";
+
+import { disableiOSZoom } from "./utils/disable-ios-zoom";
+disableiOSZoom();
+
+import { App } from "./App";
+
+import AvatarPreview from "./react-components/avatar-preview";
+import { WithHoverSound } from "./react-components/wrap-with-audio";
```
Comment (brianpeiris) [inline src/avatar.js]: We've stopped using this and we intend to remove it altogether. Let's not propagate it.

File: src/avatar.js [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+          <label className={styles.title}>
+            <span>{avatar.name}</span>
+          </label>
+          <div className={styles.attributions}>
+            {avatar.attributions && avatar.attributions.creator && <span>{`by ${avatar.attributions.creator}`}</span>}
+          </div>
+          <div className={styles.preview}>{avatar && <AvatarPreview avatarGltfUrl={avatar.gltf_url} />}</div>
+          <WithHoverSound>
+            {isSelected ? (
+              <span className={styles.selectedMessage}>This is your current avatar.</span>
```
Comment (brianpeiris) [inline src/avatar.js]: i18n?

File: src/assets/stylesheets/avatar.scss [hunk 5]
```diff
@@ -0,0 +1,96 @@
+@import 'shared';
+
+:local(.avatar-landing) {
+  position: absolute;
```
[the remark this answers is not in the document]
Comment (netpro2k) [reply src/assets/stylesheets/avatar.scss]: Just copy pasted from profile-entry-panel without much thinking, will fix

**Pull request merged**